stdlib/ysh/args-test.ysh

```diff
@@ -319,6 +319,34 @@ proc test-vs-python3-argparse {
   #assert [expected === r.stdout]
 }
 
+proc test-multi-value-flags {
+  parser (&spec) {
+    flag -f --float (List[Float])
+    flag -i --int (List[Int])
+    flag -s --str (List[Str])
+  }
+
+  var args = parseArgs(spec, :| -f 1.0 -s one -i 0 --str two --int 1 -s three |)
+
+  assert [type(args.float) === 'List']
```

Comment:
I also wonder if we should change `type(args.float)` to return a type object now, instead of a string

I think that relies on the Obj equality change we were talking about!   So both of these will work:

`type(args.float) is List`

`type(args.float) == List`

?

Something to think about